### PR TITLE
[FLINK-17371][table-runtime] Open testDefaultDecimalCasting case in DecimalTypeTest

### PR DESCRIPTION
**Pull request by JingsongLi**
Just open an ignored case.

### CODE REVIEW

**Review by leonardBang — APPROVED** [review]
LGTM

File: flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/DecimalTypeTest.scala [hunk 1]
```diff
@@ -102,10 +102,9 @@ class DecimalTypeTest extends ExpressionTestBase {
       Long.MinValue.toString)
   }
 
-  @Ignore
```
Comment (KurtYoung) [inline flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/DecimalTypeTest.scala:105]: There is an unused `Ignore` in import

**Comment by JingsongLi** [comment]
I will add more tests for the casting between decimal and other types a few days later.

**Comment by JingsongLi** [comment]
> I will add more tests for the casting between decimal and other types a few days later.

I will merge this one and create a new pull request to add cases under this JIRA.

**Comment by KurtYoung** [comment]
Sounds good to me.

**Review by KurtYoung — APPROVED** [review]

**Pull request merged**